application/src/main/resources/db/V3__Add_Mod_Audit_Log_System.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE mod_audit_log_guild_process
+(
+    guild_id                       BIGINT    NOT NULL PRIMARY KEY,
```

Comment:
Why is this `BigInt`?

Reply:
thats the type in sqlite that corresponds to long in java